definitions/marts/looker_studio_marts/ecf/ls_ecf_reg_mentor_inductions.sqlx

```diff
@@ -0,0 +1,126 @@
+config {
+    type: "table",
+    assertions: {
+        uniqueKey: ["induction_record_id"]
+    },
+    bigquery: {
+        partitionBy: "DATE(created_at)",
+        clusterBy: ["cohort"]
+    },
+    description: "This mart pulls together all Mentor induction records generated within the registration periods from 2022 onwards. It feeds into the Looker Studio 2024 ECF Registration Dashboard and informs Mentor metrics. Induction records outside of the registration window are excluded because they don't reflect activity that occurred in registration. There are three marts that inform participants registerred within the annual registration windows: ECT_inductions, Mentor_inductions and Participant_inductions. There are three as they have different fields and cannot be amalgamated into a single mart. They utilise a similar structure logic and are dependent on induction records",

```

Comment:
This table description does not contain the same distinction as the ls_ecf_reg_ect_inductions table description regarding May to October induction dates.

Reply:
Thanks for flagging! Have added it in to all relevant marts.